### PR TITLE
Adding new icons; updating and fixing existing ones;

### LINKED ISSUE #149: Icon: add new/remaining icons
A dozen or so new icons were added to the ds6 icons suite. Let's add them in 4.1.0 release.

All new:

screen shot 2018-04-18 at 3 17 27 pm

We have most of these navigation ones, but there are some new ones in there:

screen shot 2018-04-18 at 3 17 33 pm

All new:

screen shot 2018-04-18 at 3 17 36 pm

All new:

screen shot 2018-04-18 at 3 17 42 pm

Red laser is new:

screen shot 2018-04-18 at 3 19 06 pm

### LINKED ISSUE #169: SVG: round up/down viewbox attribute values
Need to go through and round out some of the values in the SVG viewboxes and make sure doing so doesn't cause any ill effect (I don't expect it will).

Example:

screen shot 2018-04-25 at 10 07 21 am

### LINKED ISSUE #179: Icon: re-encode the checkbox-unchecked background icon
I think maybe something got lost during the export from Sketch source file to base64 encoding of background svg:

screen shot 2018-05-02 at 9 58 58 am

Should look like this (excuse the size difference of screenshots here):

screen shot 2018-05-02 at 9 53 50 am

The source file seems to rely on a mask or two. We may need a new source file that uses paths instead of masks.

### PR DESCRIPTION
**Pull request by seangates**
## Description
New icons were added to the library, and some icons were updated. Also some fixes were applied ensure the background SVG (encoded versions) matched the inline SVG versions.

## References
Closes #149 
Closes #169 
Closes #179 

### Filters
image
## Navigation Support
image
## Photo Editor
image
## Reviews
image

### CODE REVIEW

File: src/svg/ds6/icons.svg [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+<svg aria-hidden="true" hidden>
```
Comment (ianmcburnie) [inline src/svg/ds6/icons.svg]: Don't think we need this change for an external SVG file. It never gets included as part of the page as far as I'm aware.

Comment (seangates) [reply src/svg/ds6/icons.svg]: Oh, my bad. That's copy pasta from the `index.html` file. Thanks for catching that.

**Comment by seangates** [comment]
@ianmcburnie Updated based on your comment. 👍

**Comment by seangates** [comment]
@ianmcburnie So ... i was able to get the flag icon back in there. I now understand the issue I was dealing with on that one. (SVGOMG was actually doing something wrong.)

Also, I added two documentation examples of all the icons mashed up next to each other as they were before. This actually helps to spot errors, too. Let me know if you think it's a good idea to leave it there, or to remove it. I'm fine either way.

**Comment by ianmcburnie** [comment]
Yep, we have a separate task for the rounding: https://github.com/eBay/skin/issues/169. We'd need to carefully see if rounding now causes any discrepancies between background and inline... it *might* do. If so, we can keep them non-rounded,

**Comment by seangates** [comment]
@PrashantAshok @ianmcburnie The rounding _does_ affect the icons, if only slightly. You'll note that the rounding for the `height` and `width` LESS variables is done to the tenth, except in one or two cases where it made a difference.

**Comment by ianmcburnie** [comment]
Ok, I closed #169. You are good to merge once conflicts are resolved.

**Pull request merged**